ci/test_wheel.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
```suggestion
# Copyright (c) 2024, NVIDIA CORPORATION.
```

This is a brand new file, so the copyright date should be 2024. In a follow-up PR, could you add the `verify-copyright` pre-commit hook here to catch and fix stuff like this?

ref: https://github.com/rapidsai/cuspatial/blob/ad2900ded3c4a855774fc47dcec9a22323f07c0b/.pre-commit-config.yaml#L48